### PR TITLE
add function for clear url from reference in path #107

### PR DESCRIPTION
**Pull request by batazor**
### Checklist:
- [x] Tests for the changes have been added (for bug fixes/features/refactoring)

Add package `github.com/stretchr/testify` for easy testing

### CODE REVIEW

File: internal/server/handler_test.go [hunk 1]
```diff
@@ -163,3 +165,22 @@ func TestPathByMaskDetect(t *testing.T) {
...
+	type test struct {
+		request  string
+		response string
+	}
+
+	tests := []test{
+		{"/user", "/user"},
+		{"/user#id", "/user"},
+		{"/user#id,password", "/user"},
+	}
```
Comment (alexander-melentyev) [inline internal/server/handler_test.go:179]: Add test when request is empty please

Comment (batazor) [reply internal/server/handler_test.go:181]: :+1:
